app/collections/Classrooms.coffee

```diff
@@ -0,0 +1,17 @@
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class Classrooms extends CocoCollection
+  model: Classroom
+  url: '/db/classroom'
+  
+  initialize: ->
+    @on 'sync', =>
+      for classroom in @models
+        classroom.capitalizeLanguageName()
```

Comment:
I do that too; But it looks like when you sync a collection, the models it contains do not get 'sync' events. I guess I could watch for `@collection.on 'sync'` in the model, too.